Makefile

```diff
@@ -34,7 +36,11 @@ deps.update:
 
 .PHONY: validate
 validate:
-	$(NPM) run validate
+	@if [ -z "${vendor-id}" ]; then \
+        $(NPM) run validate; \
+    else \
+        $(NPM) run validate -- --vendor-id $(vendor-id); \
+    fi
```

Comment:
Nit; I think you can remove the single line construct with `\` and just use multiple lines?

Reply:
i can't seem to figure out how to make it multiple lines without getting errors, so ill make it a single line